mvp.feature

```diff
@@ -40,6 +41,7 @@ Feature: MVP features for NVA
     And they see their name in the Menu
     And they see the Connect ORCID dialog
 
+  @28
```

Comment:
The title should be more informative of the whole scenario, i.e., summarize the whole story.
Right now it only describes one precondition of the scenario.

Suggestion: A user logs in while he has already been authenticated with Feide (singe sign on)

Reply:
Eksisterende features kan heller bli oppdatert i en egen PR